src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { useAccount } from 'wagmi';
+import { DataList, Tag } from '../../../../../core';
+import { addressUtils } from '../../../../utils/addressUtils';
+import { ApprovalThresholdResult } from '../approvalThresholdResult';
+import { MajorityVotingResult } from '../majorityVotingResult';
+import { ProposalDataListItemStatus } from '../proposalDataListItemStatus';
+import {
+    type IApprovalThresholdResult,
+    type IMajorityVotingResult,
+    type IProposalDataListItemStructureProps,
+} from './proposalDataListItemStructure.api';
+
+/**
+ * `ProposalDataListItemStructure` module component
+ */
+export const ProposalDataListItemStructure: React.FC<IProposalDataListItemStructureProps> = (props) => {
+    const {
+        wagmiConfig: config,
+        chainId,
+        type,
+        result,
+        date,
+        protocolUpdate,
+        publisher,
+        publisherProfileLink,
+        status,
+        summary,
+        title,
+        voted,
+        ...otherProps
+    } = props;
+
+    const { address: connectedAddress, isConnected } = useAccount({ config });
+
+    const ongoing = status === 'active' || status === 'challenged' || status === 'vetoed';
+
+    const publisherIsConnected = isConnected && connectedAddress?.toLowerCase() === publisher.address?.toLowerCase();
+    const publisherLabel = publisherIsConnected
+        ? 'You'
+        : publisher.name ?? addressUtils.shortenAddress(publisher.address as string);
+
+    return (
+        <DataList.Item className="space-y-4" {...otherProps}>
```

Comment:
We probably need to use `classNames`, otherwise this class will be replaced when adding custom classes as prop
```suggestion
        <DataList.Item className={classNames('space-y-4', className)} {...otherProps}>
```

Reply:
Good catch